new_client/src/components/auth/Signup.jsx

```diff
@@ -195,7 +197,7 @@ function Signup(props) {
 							fullWidth
 							required
 							name="graduationYear"
-							type="number"
+							type="text"
 							placeholder="Graduation Year"
 							onChange={handleChangeGraduationYear}
```

Comment:
I think we should add a limit to the number of characters that a user can enter.

Reply:
Yup, added that as an inputProp